templates/components/form/item_virtualmachine.html.twig

```diff
@@ -0,0 +1,19 @@
+{% if has_host %}
+    <h3 class='text-start'>{{ __('List of hosts') }}</h3>
+{% else %}
+    <h3 class='text-start mt-5'>{{ __('List of virtualized environments') }}</h3>
+    {% if canedit %}
+        <div class='center firstbloc'>
+            <a class='btn btn-primary' href='{{ vmformurl }}'>
+                {{ __('Add a virtual machine') }}
+            </a>
+        </div>
+    {% endif %}
+
+    {% if not has_vm %}
+        <div class='alert alert-info'>
```

Comment:
Should use the macro from `templates/components/alerts_macros.html.twig` to standardize the display of alerts.